CHANGELOG.md

```diff
@@ -1,6 +1,28 @@
 ## Unreleased
 
-* [FEATURE] Add `-kubelet-endpointslice` argument to support `EndpointSlice` for the kubelet controller.
+## 0.77.0 / 2024-09-18
+
```

Comment:
I think we can remove this extra line

Reply:
we need blank line after heading for markdown